scripts/components/HUD/Buttons/Button/Button.scss

```
@@ -37,7 +37,7 @@
     height: 0.75em;
     position: absolute;
   }
-  &:hover .tooltip {
+  &:hover .tooltip, &:focus-within .tooltip {
     visibility: visible;
     opacity: 1;
     transition-delay: 0s;
```

This is a really nice touch, but this will break `:hover` styling on browser that does not support `:focus-within`. Consider duplicating the styling (and perhaps comment it well enough for people not to re-do it), or create a `mixin` that is included in both selectors.

```suggestion
  &:hover .tooltip {
    visibility: visible;
    opacity: 1;
    transition-delay: 0s;
  }
  
  &:focus-within .tooltip {
    visibility: visible;
    opacity: 1;
    transition-delay: 0s;  
```

or

```suggestion
  @mixin tooltip-styling {
    visibility: visible;
    opacity: 1;
    transition-delay: 0s;
  }

  &:hover .tooltip {
    @include tooltip-styling;
  }
  
  &:focus-within .tooltip {
    @include tooltip-styling;
```